deploy/helm/charts/values.yaml

```diff
@@ -70,14 +70,21 @@ serviceAccount:
   name:
 
 classes:
```

Comment:
Should we use `storageClasses` instead of just `classes` here?

Reply:
then there will be repeatition of storageclass in the values